pages/community/gsoc.js

```diff
@@ -0,0 +1,100 @@
+              <br></br>
+              We mentored 5 mentees. All of them successfully completed GSoC. As a result, all of them became members of our Technical Steering Committee and also presented their solutions at AsyncAPI Conference 2021:
+            
+              <ul>
+                <li>GSoC: Writing the spec document without knowing the specification - Elegbede Azeez W., Individual C.</li>
+                <li>GSoC: Visualise your defined EDA using Cupid - Arjun Garg, Individual Contributor</li>
+                <li>GSoC: Make your AsyncAPI document shorter with Optimizer - Khuda Dad Nomani, Individual Contributor</li>
+                <li>GSoC: Generating diffs using AsyncAPI Diff - Aayush Sahu, Individual Contributor</li>
+                <li>Nektarios that created https://github.com/asyncapi/simulator could not, unfortunately, join the conference</li>
+              </ul>
+        
+              We want to repeat history on an even larger scale in 2022.  
+              </Paragraph>
+          </div>
+
+          <div className="relative mb-8 lg:mt-8">
+              <Heading level="h2" typeStyle="heading-md-semibold">
+              What about 2022?
+              </Heading> 
+              <Paragraph className="mt-4 lg:pr-4">
+              We already shared some of the details in our live stream dedicated to contributors. We already counted about 8 mentors (AsyncAPI maintainers) that want to join in this GSoC edition and help. The number will grow for sure.
+              </Paragraph>
+          </div>
+
+          <div className="relative mb-8 lg:mt-8">
+              <Heading level="h3" typeStyle="heading-md-semibold">
+              How we work on proposals
+              </Heading> 
+              <Heading level="h4" typeStyle="heading-md-semibold">
+              All on GitHub
+              </Heading> 
+              <Paragraph className="mt-4 lg:pr-4">
+              AsyncAPI Initiative is a community-driven organization that puts radical transparency at the core of its values.
+              <br></br>
+              This means all the work on proposals should take place in GitHub and be fully transparent.
+              <br></br>
+              Another good reason for this approach is that it is easy to identify duplicates and proposals that have more than one candidate. We do not want to reject candidates just because there are too many for one proposal. Let us discover it sooner and make sure we can enable as many people as possible.
+              </Paragraph>
+          </div>
+
+          <div className="relative mb-8 lg:mt-8">
+              <Heading level="h4" typeStyle="heading-md-semibold">
+              Processing Proposals 
+              </Heading> 
+              <Paragraph className="mt-4 lg:pr-4">
+              Proposals can be submitted either as GitHub Discussion or GitHub Issue in AsyncAPI GitHub organization.
+              <br></br>
```

Comment:
If you are making the points, better you make them using `<ul>` tag instead of using `<br>` tag everywhere. Also, `<br>` is a self-closing tag, so it should be used like this `<br />`, not like this `<br></br>`.